src/contracts/GPv2AccessControl.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: LGPL-3.0-or-newer
+pragma solidity ^0.6.12;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/EnumerableSet.sol";
+
+/// @title Gnosis Protocol v2 Access Control Contract
+/// @author Gnosis Developers
+contract GPv2AccessControl is Ownable {
+    using EnumerableSet for EnumerableSet.AddressSet;
+
+    EnumerableSet.AddressSet private solvers;
+
+    function addSolver(address solverAddress) public {
```

Comment:
I am okay with keeping it so general open in the first PR

Reply:
Shouldn't this be onlyOwner for now? Or do you want to keep it uncontrolled in the beginning?

Reply:
The next PR is intended to include a transfer function for the purpose of staking GNO. So essentially this is meant to remain public once we introduce the slashing/staking. Sorry if I forgot to mention this in the description.

Reply:
I missed this decision. We are going to launch with staking in place? As far as I understood we were going to start with a DAO owned list.

Reply:
Right, I do recall this being discussed. If that is the direction we plan to go, then we may want to make this method `onlyOwner` but would also want to implement something that would allow us to change this later. I guess this would coincide with the concept of upgradeability.

Reply:
There was no decision! But likely we would prefer to have a permissionless system and hence staking. I am fine with keeping it as it is or using `onlyOwner` until a decision is made.

Reply:
That's good. Should we also implement the possibility to remove solvers in this PR as well? 

After this is made and possible decision on storage accessible I suppose this is ready for merge.

Reply:
At this moment, the question remains open regarding the introduction of a method `removeSolver` and whether we should limit this/these methods to `onlyOwner` for the time being. 

@josojo

Reply:
I get the impression that including `removeSolver` is probably a good idea and there is no harm in making this `onlyOwner` for the time being.

Reply:
Ok, I have implemented and tested removeSolver as well as restricting both add and remove solver to onlyOwner. 💯

Reply:
completely agree!